webpack/ForemanTasks/Components/TasksDashboard/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -0,0 +1,138 @@
+      (focusedOn.error && (focusedOn.error.total || focusedOn.error.last)));
+  render() {
+    const {
+      className,
+      error,
+      warning,
+      success,
+      timePeriod,
+      focusedOn,
+      onTitleClick,
+    } = this.props;
+    const data = { error, warning, success };
+
+    return (
+      <Card
+        className={classNames(
+          'tasks-donut-card',
+          'stopped-tasks-card',
+          className,
+          {
+            'selected-tasks-card': this.shouleBeSelected(focusedOn),
+          }
+        )}
+      >
+        <Card.Title onClick={onTitleClick}>{__('Stopped')}</Card.Title>
+        <Card.Body>
+          <table className="table table-titles">
+            <tbody>
+              <tr>
+                <td />
+                <td>Total</td>
+                <td>{timePeriod}</td>
+              </tr>
+            </tbody>
+          </table>
+          <table className="table table-bordered table-striped">
+            <tbody>
+              {Object.keys(data).map((item, index) => (
+                <tr className={`${item}-row`} key={index}>
+                  <td>{item[0].toUpperCase() + item.slice(1)}</td>
+                  <td
+                    className={this.getTableItemClassName(item, 'total')}
+                    onClick={data[item].total.onClick}
+                  >
+                    {data[item].total.value}
+                  </td>
+                  <td
+                    className={this.getTableItemClassName(item, 'last')}
+                    onClick={data[item].last.onClick}
+                  >
+                    {data[item].last.value}
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </Card.Body>
+      </Card>
+    );
+  }
+}
+
+StoppedTasksCard.propTypes = {
+  className: PropTypes.string,
+  focusedOn: PropTypes.shape({
+    normal: PropTypes.bool,
+    error: PropTypes.shape({
+      total: PropTypes.bool,
+      last: PropTypes.bool,
+    }),
+    warning: PropTypes.shape({
+      total: PropTypes.bool,
+      last: PropTypes.bool,
+    }),
+    success: PropTypes.shape({
+      total: PropTypes.bool,
+      last: PropTypes.bool,
+    }),
+  }),
+  onTitleClick: PropTypes.func,
+  error: PropTypes.shape({
+    total: PropTypes.shape({
+      value: PropTypes.number,
+      onClick: PropTypes.func,
+    }),
+    last: PropTypes.shape({ value: PropTypes.number, onClick: PropTypes.func }),
+  }),
+  warning: PropTypes.shape({
+    total: PropTypes.shape({
+      value: PropTypes.number,
+      onClick: PropTypes.func,
+    }),
+    last: PropTypes.shape({ value: PropTypes.number, onClick: PropTypes.func }),
+  }),
+  success: PropTypes.shape({
+    total: PropTypes.shape({
+      value: PropTypes.number,
+      onClick: PropTypes.func,
+    }),
+    last: PropTypes.shape({ value: PropTypes.number, onClick: PropTypes.func }),
+  }),
+  timePeriod: PropTypes.string,
```

Comment:
Can you resort the propTypes so the one with literal values will be at the top of the list?

It makes it easier to distinguish them.